lua/onedarkpro/highlights/filetypes/php.lua

```diff
@@ -0,0 +1,19 @@
+local M = {}
+
+---Get the highlight groups for the filetype
+---@param theme table
+---@return table
+function M.groups(theme)
+    local config = require("onedarkpro.config").init()
+
+    return {
+        ["@method.php"] = { fg = theme.palette.blue, style = config.options.bold },
+        ["@method.call.php"] = { link = "@method.php" },
+        ["@function.builtin.php"] = { fg = theme.palette.cyan },
+        ["@keyword.php"] = { fg = theme.palette.purple, style = "NONE" },
```

Comment:
@olimorris Is this the right way to handle the `style`? Setting it to `NONE` gets things closer to how vscode displays PHP for me, but should this be left to how the user has `config.styles.keywords` set?

Reply:
Brilliant thank you. I should probably be more explicit in the contributing guidelines and ask everyone to turn on bolds and italics in VS Code and match them in the filetype highlights with the `style = config.options.*`.

Reply:
@olimorris Sorry, I'm not fully sure what the best practice is here.

There are three possible ways to set the style here:

1. Explicit `"NONE"` (force it to match the vscode theme)
2. Follow `config.options.bold`
3. Follow `config.styles.keywords`

I'm guessing the second option is only for when the vscode theme, when its bold option is enabled, makes the item bold, so we probably would not want to use it here.

But what about the third option?

Reply:
Sure. Take a look at `public`, `function`, and `return` below. Left is neovim.

![image](https://user-images.githubusercontent.com/524994/196227231-6924a665-c66a-432c-ba69-45277c0ec6da.png)

I have `keywords = "italic"` in my config (which I forgot when I made this PR), which is why they are italicized in neovim. I overrode that in the highlight in this PR, but I probably shouldn't do that. It should probably respect `config.styles.keywords`, and leave it up to the user to decide if they want to change `config.styles.keywords` or add highlights that override it for specific languages. I wasn't sure, though, so I wanted to check with you.

Reply:
I agree; let's make it respect `config.styles.keywords`. I'll update the contributing guide too. Thanks for pointing this out.